src/cluster_legacy.c

```diff
@@ -211,7 +217,9 @@ typedef enum {
     af_human_nodename,
     af_tcp_port,
     af_tls_port,
-    af_count,
+    af_client_ipv4,
+    af_client_ipv6,
```

Comment:
Align with the setting as well?
```suggestion
    af_announce_client_ipv4,
    af_announce_client_ipv6,
```